next.config.js

```diff
@@ -10,7 +10,10 @@ const withPWA = require('next-pwa');
 const runtimeCaching = require('next-pwa/cache');
 
 const nextConfig = withPWA({
-  swcMinify: true,
+  swcMinify: true, // opt-in to use the Next.js compiler for minification
+  experimental: {
+    forceSwcTransforms: true, // to fix - `Disabled SWC because of custom Babel configuration ".babelrc"` (https://nextjs.org/docs/messages/swc-disabled)
```

Comment:
@hussamkhatib We actually don't need this option because the only thing we were using the babel config was for the `inline-react-svg` plugin, but we already have `@svgr/webpack` for that added to the webpack plugins.

So you can go ahead and delete the `.babelrc` file and remove this option from the next config. That way Next.js would use [swc by default](https://nextjs.org/docs/upgrading#swc-replacing-babel), when a babel config is not present.

Reply:
It was added by @gupta-ji6 .
I have made the changes as It's not required.